src/treelearner/serial_tree_learner.cpp

```diff
@@ -346,9 +346,9 @@ void SerialTreeLearner::FindBestSplits(const Tree* tree, const std::set<int>* fo
   } else {
     ConstructHistograms(is_feature_used, use_subtract);
   }
-#else
+#else  // USE_CUDA
   ConstructHistograms(is_feature_used, use_subtract);
-#endif
+#endif  // USE_CUDA
```

Comment:
could these changes be made in a separate PR? That would remove this file from the diff entirely.

Reply:
Resovled via ba22deb